webapp/lib/charts.dart

```diff
@@ -145,7 +145,7 @@ class DailyTimeseriesLineChartView {
     chart = chartjs.Chart(canvas.getContext('2d'), chartConfig);
   }
 
-  void updateChart([List<Map<DateTime, int>> updatedCountsAtTimestampList, String timeScaleUnit = 'day']) {
+  void updateChart([List<Map<DateTime, dynamic>> updatedCountsAtTimestampList, String timeScaleUnit = 'day']) {
```

Comment:
would it make sense to replace dynamic here with `num`? I'm assuming the change is needed because now we have doubles as well as ints?

Reply:
Hadn't thought of that. It's better than dynamic. Changed it